src/AzureIoTHub.Portal.Infrastructure/Managers/AwsDeviceModelImageManager.cs

```diff
@@ -0,0 +1,224 @@
+
+            };
+
+            var putObjectResponse = await this.s3Client.PutObjectAsync(putObjectRequest);
+
+            if (putObjectResponse.HttpStatusCode == System.Net.HttpStatusCode.OK)
+            {
+                //Images on S3 are publicly accessible and read-only 
+                var putAclRequest = new PutACLRequest
+                {
+                    BucketName = this.configHandler.AWSBucketName,
+                    Key = deviceModelId,
+                    CannedACL = S3CannedACL.PublicRead // Set the object's ACL to public read
+                };
+                var putACLResponse = await this.s3Client.PutACLAsync(putAclRequest);
+
+                return putACLResponse.HttpStatusCode == System.Net.HttpStatusCode.OK
+                    ? ComputeImageUrl(deviceModelId)
+                    : throw new InternalServerErrorException("Error by setting the image access to public and read-only");
+            }
+            else
+            {
+                throw new InternalServerErrorException("Error by uploading the image in S3 Storage");
+            }
+
+        }
+
+        public async Task InitializeDefaultImageBlob()
+        {
+
+            this.logger.LogInformation($"Initializing default Image to AWS S3 storage");
+
+            var putObjectRequest = new PutObjectRequest
+            {
+                BucketName = this.configHandler.AWSBucketName,
+                Key = this.imageOptions.Value.DefaultImageName,
+                FilePath = $"../Resources/{this.imageOptions.Value.DefaultImageName}",
+                ContentType = "image/*", // image content type
+                Headers = {CacheControl = $"max-age={this.configHandler.StorageAccountDeviceModelImageMaxAge}, must-revalidate" }
+
+            };
+
+            var putObjectResponse = await this.s3Client.PutObjectAsync(putObjectRequest);
+
+            if (putObjectResponse.HttpStatusCode == System.Net.HttpStatusCode.OK)
+            {
+                //Images on S3 are publicly accessible and read-only 
+                var putAclRequest = new PutACLRequest
+                {
+                    BucketName = this.configHandler.AWSBucketName,
+                    Key = this.imageOptions.Value.DefaultImageName,
+                    CannedACL = S3CannedACL.PublicRead // Set the object's ACL to public read
+                };
+                var putACLResponse = await this.s3Client.PutACLAsync(putAclRequest);
+
+                if (putACLResponse.HttpStatusCode != System.Net.HttpStatusCode.OK)
+                {
+                    throw new InternalServerErrorException("Error by setting the image access to public and read-only");
+                }
+            }
+            else
+            {
+                throw new InternalServerErrorException("Error by uploading the image in S3 Storage");
+            }
+
+        }
+
+        public async Task SyncImagesCacheControl()
```

Comment:
The method SyncImagesCacheControl is never called. 

On Azure we used the SyncImagesCacheControl on the class DeviceModelImageManager the update the CacheControl  of existing images on stratup. IMO we don't need an implementation of this mehod for AWS because new images will processed by the method SetDefaultImageToModel